src/AssetManager.php

```diff
@@ -316,4 +316,15 @@ public function extractGroupDepsNamesFromDepsHandles($depsHandles, $group, $type
 
         return array_keys($requiredGroups);
     }
+
+    public function getDistinctGroups(string $dependencyType) {
```

Comment:
```suggestion
    public function getDistinctGroupsDependencies(string $dependencyType) {
```
+ ajouter un commentaire.